spa/src/components/portal/ContributionsList/ContributionDetail/BillingHistory/BillingHistory.tsx

```diff
@@ -21,9 +25,18 @@ const PAYMENT_STATUS_NAMES: Record<PortalContributionPayment['status'], string>
 
 const dateFormatter = Intl.DateTimeFormat(undefined, { day: 'numeric', month: 'numeric', year: 'numeric' });
 
-export function BillingHistory({ disabled, payments }: BillingHistoryProps) {
+export function BillingHistory({ disabled, payments, onSendEmailReceipt }: BillingHistoryProps) {
+  const { page } = usePortal();
+  const sendNreEmail = page?.revenue_program?.organization?.send_receipt_email_via_nre;
+
   return (
-    <DetailSection disabled={disabled} title="Billing History">
+    <DetailSection
+      disabled={disabled}
+      title="Billing History"
+      {...(sendNreEmail && {
+        controls: <SectionControlButton onClick={onSendEmailReceipt}>Resend receipt</SectionControlButton>
+      })}
```

Comment:
```suggestion
      controls={
        sendNreEmail && <SectionControlButton onClick={onSendEmailReceipt}>Resend receipt</SectionControlButton>
      }
```

A little more concise.